### PR TITLE
KAFKA-16551: add integration test for ClusterTool

### PR DESCRIPTION
**Pull request by FrankYang0529**
Add `testClusterId` and `testUnregister`.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: tools/src/test/java/org/apache/kafka/tools/ClusterToolTest.java [hunk 1]
```diff
@@ -16,20 +16,67 @@
...
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @Timeout(value = 60)
+@ExtendWith(value = ClusterTestExtensions.class)
 public class ClusterToolTest {
 
+    @ClusterTest
+    public void testClusterId(ClusterInstance clusterInstance) {
+        String output = ToolsTestUtils.captureStandardOut(() -> {
```
Comment (chia7712) [inline tools/src/test/java/org/apache/kafka/tools/ClusterToolTest.java]:
```java
        String output = ToolsTestUtils.captureStandardOut(() ->
            assertDoesNotThrow(() -> ClusterTool.execute("cluster-id", "--bootstrap-server", clusterInstance.bootstrapServers())));
```

File: tools/src/test/java/org/apache/kafka/tools/ClusterToolTest.java [hunk 2]
```diff
@@ -16,20 +16,67 @@
...
+            } catch (Exception e) {
+                throw new RuntimeException(e);
+            }
+        });
+        assertTrue(output.contains("Cluster ID: " + clusterInstance.clusterId()));
+    }
+
+    @ClusterTest(brokers = 3)
+    public void testUnregister(ClusterInstance clusterInstance) {
+        Integer brokerId;
```
Comment (chia7712) [inline tools/src/test/java/org/apache/kafka/tools/ClusterToolTest.java]: `int`

File: tools/src/test/java/org/apache/kafka/tools/ClusterToolTest.java [hunk 3]
```diff
@@ -16,20 +16,67 @@
...
+            brokerId = assertDoesNotThrow(() -> clusterInstance.brokerIds().stream().findFirst().get());
+        } else {
+            Set<Integer> brokerIds = clusterInstance.brokerIds();
+            brokerIds.removeAll(clusterInstance.controllerIds());
+            brokerId = assertDoesNotThrow(() -> brokerIds.stream().findFirst().get());
+        }
+        clusterInstance.shutdownBroker(brokerId);
+        String output = ToolsTestUtils.captureStandardOut(() -> {
+            try {
+                ClusterTool.execute("unregister", "--bootstrap-server", clusterInstance.bootstrapServers(), "--id", brokerId.toString());
```
Comment (chia7712) [inline tools/src/test/java/org/apache/kafka/tools/ClusterToolTest.java]: ditto. consider using `assertDoesNotThrow`

**Review by chia7712 — COMMENTED** [review]
@FrankYang0529 thanks for this patch!

**Comment by FrankYang0529** [comment]
Hi @chia7712, thanks for the review. I addressed all comments.

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request merged**